package.json

```diff
@@ -17,7 +17,8 @@
     "coverage:html": "nyc report --reporter=html",
     "format": "prettier --loglevel silent --write .; prettierx --write .",
     "lint": "eslint .",
-    "lint:fix": "eslint . --fix"
+    "lint:fix": "eslint . --fix",
+    "prepublishOnly": "npm run test:prod"
```

Comment:
It should be `npm run build:prod`. Tests are already run in the dependent job.

Reply:
Okay

Reply:
Personally I find it a bit dangerous to not test before publishing and just rely on the GitHub Actions workflow script, but maybe we will never create a release manually